examples/server/tests/unit/test_tool_call.py

```diff
@@ -306,6 +312,9 @@ def test_completion_without_tool_call_slow(template_name: str, n_predict: int, t
     ("bartowski/Phi-3.5-mini-instruct-GGUF:Q4_K_M",      None),
     ("bartowski/Phi-3.5-mini-instruct-GGUF:Q4_K_M",      "chatml"),
 
```

Comment:
Could you throw in a default (generic) test case for hello_world, weather and calc_result? Just in case their default template changes and something goes boom.

```
("bartowski/microsoft_Phi-4-mini-instruct-GGUF:Q4_K_M",      None),
```